pkg/scheduler/core/extender.go

```diff
@@ -31,7 +31,7 @@ import (
 	extenderv1 "k8s.io/kube-scheduler/extender/v1"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/apis/config"
 	"k8s.io/kubernetes/pkg/scheduler/listers"
-	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
+	schedulertypes "k8s.io/kubernetes/pkg/scheduler/types"
```

Comment:
we don't need this alias in a `pkg/scheduler` package. Same for the rest of the imports that don't have collisions and are within `pkg/scheduler`.

Reply:
This is a significant refactor, which I tried to automate, it will be too much work to select the places where this conflicts and where it doesn't, and the return is not obvious. The alias type is mostly used to refer to the apimachinary one. We also used schedulerlisters, so this is not diverging too much.